packages/disputer/index.js

```diff
@@ -167,40 +172,41 @@ async function Poll(callback) {
       );
     }
 
-    // Default to 1 minute delay. If set to 0 in env variables then the script will exit after full execution.
-    const pollingDelay = process.env.POLLING_DELAY ? Number(process.env.POLLING_DELAY) : 60;
-
-    // Default to 5 re-tries on error within the execution loop.
-    const errorRetries = process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5;
-
-    // Default to 10 seconds in between error re-tries.
-    const errorRetriesTimeout = process.env.ERROR_RETRIES__TIMEOUT ? Number(process.env.ERROR_RETRIES__TIMEOUT) : 10;
-
-    // Read price feed configuration from an environment variable. This can be a crypto watch, medianizer or uniswap
-    // price feed Config defines the exchanges to use. If not provided then the bot will try and infer a price feed
-    // from the EMP_ADDRESS. EG with medianizer: {"type":"medianizer","pair":"ethbtc",
-    // "lookback":7200, "minTimeBetweenUpdates":60,"medianizedFeeds":[{"type":"cryptowatch","exchange":"coinbase-pro"},
-    // {"type":"cryptowatch","exchange":"binance"}]}
-    const priceFeedConfig = process.env.PRICE_FEED_CONFIG ? JSON.parse(process.env.PRICE_FEED_CONFIG) : null;
-
-    // If there is a disputer config, add it. Else, set to null. This config contains disputeDelay and txnGasLimit. EG:
-    // {"disputeDelay":60,"txnGasLimit":9000000}
-    const disputerConfig = process.env.DISPUTER_CONFIG ? JSON.parse(process.env.DISPUTER_CONFIG) : null;
+    // This object is spread when calling the `run` function below. It relies on the object enumeration order and must
+    // match the order of parameters defined in the`run` function.
+    const executionParameters = {
```

Comment:
+1. OOC was `executionParameters` used in liquidator as well? This is much cleaner

Reply:
yup, it is in the current master [version](https://github.com/UMAprotocol/protocol/blob/c50aa4765b3a762f74c28ad9f0ae7da5bad1cac8/packages/liquidator/index.js#L211). For now this object is spread to be normal param inputs. In a follow on an [issue](https://github.com/UMAprotocol/protocol/issues/1869) we'll refactor this to pass in the object as named params.